src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -125,7 +125,7 @@ internal void InitializeSourceInfo(int iSkip, bool fNeedFileInfo, Exception exce
                 if (s_symbolsMethodInfo == null)
                 {
                     s_symbolsType = Type.GetType(
-                        "System.Diagnostics.StackTraceSymbols, System.Diagnostics.StackTrace, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a",
+                        "System.Diagnostics.StackTraceSymbols, System.Diagnostics.StackTrace, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a",
```

Comment:
As long as Roslyn isn't removed from the shared framework (though I've heard it has been discussed), it shouldn't matter.  With this change coreclr doesn't depend on the version of S.D.ST because even if Roslyn references a newer version, the coreclr support will still work.

/cc: @weshaggard